org.polymc.PolyMC.metainfo.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally, this should be maintained upstream. Not a blocker, though.

Reply:
I will try to get it upstream but for now it can stay I think. Problem is stuff like org.polymc.PolyMC normally would just be PolyMC

Reply:
FYI the standards for desktop, metainfo and icon files mention files should use reverse DNS like here, it's just that nothing has ever enforced this. So it's fine for PolyMC itself to use rDNS as well.

Reply:
> FYI the standards for desktop, metainfo and icon files mention files should use reverse DNS like here, it's just that nothing has ever enforced this. So it's fine for PolyMC itself to use rDNS as well.

Ok, submitted [#13](https://github.com/PolyMC/PolyMC/issues/13) and [#12](https://github.com/PolyMC/PolyMC/issues/12) to upstream.